README.md

```diff
@@ -233,16 +233,17 @@ The learn more about the possibilities for postprocessing check out our [helper
 
 ## Building / Releasing
 
-1. `npm run dist` and commit the built output (yes, you read that right)
-2. Bump whatever you want to bump in the `package.json` version field
-3. Merge `main` into `vMAJOR` branch. `git checkout vMAJOR && git merge main`
+1. Set the `NODE_OPTIONS` environment variale to `--openssl-legacy-provider`
```

Comment:
might it make more sense to bake this into the `dist` command? https://github.com/githubocto/flat/blob/main/package.json#LL10C14-L10C14

Reply:
Yes! I will update and test.